lms/static/certificates/sass/_ltr.scss

```diff
@@ -0,0 +1,23 @@
+// ------------------------------
```

Comment:
I thought that this partial could be imported directly from the UXPL. Why do we need our own copy? I was hoping that references to bi-app would be kept inside the UXPL.